packages/zendesk-adapter/src/config.ts

```diff
@@ -2562,11 +2562,7 @@ export const DEFAULT_TYPES: ZendeskApiConfig['types'] = {
   custom_object_fields: {
     request: {
       url: '/api/v2/custom_objects/{custom_object_key}/fields',
-      queryParams: {
-        ...DEFAULT_QUERY_PARAMS,
-        // Some fields are 'standard' and are not custom by the user, and won't be returned unless explicitly requested
-        include_standard_fields: 'true',
```

Comment:
It is created automatically, and when i tried to create an element with these i received an error that they cant be created because they already exists